__init__.py

```diff
@@ -504,22 +504,31 @@ def play_song(self, message):
         artist = message.data.get('artist')
         album = message.data.get('album')
 
-        # workaround for Padatious training, as the most generic "play {track}" is taking precedence over the play_something
-        # and play_playlist rules
+        # workaround for Padatious training, as the most generic "play {track}"
+        # is taking precedence over the play_something and play_playlist rules
         if song and not album:
-            m = re.match(r'^play( the| my)? playlist (?P<playlist>[\w\s]+?)$' , message.data['utterance'], re.M|re.I)
+            if song == 'spotify':
+                self.continue_current_playlist(message)
+            LOG.info(self.translate('playlist_regex'))
+            m = re.match(self.translate('playlist_regex'),
+                         message.data['utterance'], re.M | re.I)
```

Comment:
I don't know if this is Github or not, but it looks like lines 512 and 513 aren't in the branch created at 510.

Reply:
Ah yes, there should be a return statement between 511 and 512